sky/backends/cloud_vm_ray_backend.py

```diff
@@ -3004,6 +3007,12 @@ def _execute_task_n_nodes(self, handle: ResourceHandle, task: task_lib.Task,
         codegen.add_gang_scheduling_placement_group(num_actual_nodes,
                                                     accelerator_dict)
 
+        ip_list = backend_utils.get_node_ips(handle.cluster_yaml,
```

Comment:
Before L3010, document whether get_node_ips() guarantees the stableness, or whether L3014 hard-codes it once hence becoming stable? Not clear at first glance.

What happens if this func is called twice; is `ip_list` guaranteed to be the same? E.g., two `sky exec -c .. --num-nodes=<n> ...`.

Reply:
Ran some experiments: it looks like `ray get-worker-ips` (which is used by `get_node_ips`) does return the same order each time. However, since this does not seem to be a documented feature, I think it makes sense to sort the worker node ips retrieved to add this guarantee.